packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1561,8 +1561,7 @@ class _ActionButtonParentDataWidget
   }
 
   @override
-  Type get debugTypicalAncestorWidgetClass =>
-      _CupertinoDialogActionsRenderWidget;
+  String get debugTypicalAncestorWidgetClass => '_CupertinoDialogActionsRenderWidget';
```

Comment:
You could use `'$SomeClass'` instead for all these usages?

Reply:
Could do! I don't know if that would be reliably followed going forward since the type is just a String